silo/gviews_v4.py

```diff
@@ -409,24 +408,36 @@ def export_to_gsheet_helper(user, spreadsheet_id, silo_id, query, headers):
                                 and header not in other_title:
                             repeat_headers.append(header)
                     else:
-                        values.append({"userEnteredValue": {"stringValue":
-                                                                ""}})
+                        values.append({"userEnteredValue":
+                                           {"stringValue": ""}})
                 else:
-                    values.append({"userEnteredValue": {
-                        "stringValue": smart_text(row[header])}})
+                    if not (header == '_id'):
+                        values.append({"userEnteredValue": {
+                            "stringValue": smart_text(row[header])}})
+                    else:
+                        values.append({"userEnteredValue": {
+                            "stringValue": smart_text(row[header]['$oid'])}})
+
             except KeyError:
                 values.append({"userEnteredValue": {"stringValue": ""}})
         rows.append({"values": values})
 
     # prepare column names as a header row in spreadsheet
     values = []
     for header in headers:
-        values.append({
-                      "userEnteredValue": {"stringValue": header},
-                      'userEnteredFormat':
-                          {'backgroundColor':
-                               {'red':0.5,'green':0.5, 'blue': 0.5}}
-                      })
+        if not (header == '_id'):
+            values.append({
+                          "userEnteredValue": {"stringValue": header},
+                          'userEnteredFormat':
+                              {'backgroundColor':
+                                   {'red':0.5,'green':0.5, 'blue': 0.5}}
+                          })
+        else:
+            values.append({
+                        "userEnteredValue": {"stringValue": 'id'},
+                        'userEnteredFormat':
+                            {'backgroundColor':
+                                 {'red': 0.5, 'green': 0.5, 'blue': 0.5}}})
```

Comment:
Why did you not reuse the following code instead of duplicating it? E.g, use a variable for the values.
```
values.append({
    "userEnteredValue": {"stringValue": header},
    'userEnteredFormat':
            {'backgroundColor':
                    {'red': 0.5, 'green': 0.5, 'blue': 0.5}}})
```